themes/default/content/docs/intro/deployments/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title_tag: "Pulumi Deployments"
+title: "Pulumi Deployments"
+meta_desc: Pulumi Deployments is the fast way for individuals, teams, enterprises, and platforms to go from code to cloud.
+menu:
+  intro:
+    identifier: deployments
+    weight: 6
+
+aliases: ["/docs/intro/pulumi-service/deployments/"]
+---
+
+Pulumi Deployments is a fully managed platform for managing infrastructure at scale. Whether you are just getting started in the cloud and want the easiest path to set up secure CI/CD, or are a platform engineer building a self-service platform to support thousands of engineers, Pulumi Deployments is the fastest way to go from code to cloud. We provide the building blocks that enable you to scale your cloud footprint without scaling your headcount, with tools that are designed to power public and private cloud platforms, APIs, and self-service infrastructure portals.
+
+It provides:
+
+- **Compute** - run infrastructure deployments and workloads (ie `pulumi up`) on fully managed, secure, and elastic compute
+- **Configuration** - A simple, centralized format for specifying everything you need to deploy your infrastructure at the stack level. Source code, cloud credentials, OpenID Connect (OIDC), environment variables, build prerequisites, and dependencies are bundled in an atomic unit
+- **Composition** - Trigger deployments via our REST API, click to deploy in the Pulumi Service UI, Git Push to Deploy, Remote Automation API, and other programmatic building blocks that you can compose to build custom cloud automation
+
+While Pulumi Deployments has some overlap in functionality offered by a traditional CI/CD system (such as GitHub Actions) it is far more flexible and does not have the constraints and limitations of a CI/CD system. When you configure your infrastructure for Deployments, you get to choose what triggers will cause your stack to update. You may choose a `git push` based workflow, as you get with many CI/CD systems, or you may trigger deployments via the REST API, or you might choose multiple triggers.
```

Comment:
I'm not sure I agree with this and think it's dangerous territory. I don't think we can say we're more flexible than say, github actions, but we can say we are better at infrastructure deployment.

No concrete ideas for improvement off the cuff, but I'd lean towards not even bringing up CI/CD unless we think we can't avoid it.

Reply:
We are more flexible than github actions. We have a REST API to trigger a workflow run. That API can be called from within lambda, ECS, temporal, or any other higher level software system you want to build. 

I think we have to bring up CI/CD. Customers very often ask us how this compares to github actions and why they would use one over the other. 

Open to adjustments here if you have ideas.

Reply:
Notes from Lee: 

Maybe more choice, more modular, more composable. Because this is a first class experience designed from the ground up for infrastructure. 

YOu have three parts:

- trigger
- configuration
- program

choose your own adventure

Reply:
The word flexibility may be a trigger for some people.